src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -90,6 +92,15 @@ public async Task ConfigRevisions()
                         }
                     }
                 });
+
+            if (enforce)
```

Comment:
I do not like this. I think it would be better to remove this and just have a separate endpoint to do that.

Reply:
removed